### PR TITLE
Fixed Typos

### PR DESCRIPTION
**Pull request by shubhamgoel90**

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -11,7 +11,7 @@ for this branch are our priority.
 
 ### Reporting an Issue
 
-- Make sure you test against the latest version (of the highlyunstable branch). It is possible we 
+- Make sure you test against the latest version (of the highly unstable branch). It is possible we
```
Comment (orensbruli) [inline CONTRIBUTING.md]:
```suggestion
- Make sure you test against the latest version (of the development branch). It is possible we 
```

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -40,7 +40,7 @@ If you need to sync commits from `robocomp/robocomp` to push your own changes, f
 be sure to update the `Readme.md` installation tutorial to include your changes.
 
 - If you plan to make big changes, ensure you are not breaking Robocomp's core functionalities.
-Even through Travis tests the build, you could break an internal function. 
+Even though Travis tests the build, you could break an internal function.
```
Comment (orensbruli) [inline CONTRIBUTING.md]:
```suggestion
Even though CI tests the build, you could break an internal function. 
```

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -40,7 +40,7 @@ If you need to sync commits from `robocomp/robocomp` to push your own changes, f
 be sure to update the `Readme.md` installation tutorial to include your changes.
 
 - If you plan to make big changes, ensure you are not breaking Robocomp's core functionalities.
-Even through Travis tests the build, you could break an internal function. 
+Even though Travis tests the build, you could break an internal function. 
 Please ensure you test new features locally at first.
```
Comment (orensbruli) [inline CONTRIBUTING.md]:
```suggestion
Please ensure you test new features locally at first and add some unit tests for the new functionalities.
```

Comment (orensbruli) [reply CONTRIBUTING.md]: Thank you for your contribution and for pointing to this piece of the documentation.

Comment (shubhamgoel90) [reply CONTRIBUTING.md]: Welcome, @orensbruli.

**Pull request merged**